bson.cpp

```diff
@@ -424,6 +428,91 @@ void VariantToBsonConverter::_convertSerializable(bson_t *bson, const char *key,
 }
 /* }}} */
 
+/* {{{ MongoDriver\Driver\ReadConcern */
+void VariantToBsonConverter::_convertReadConcern(bson_t *bson, const char *key, Object v)
+{
+	MongoDBDriverReadConcernData* data = Native::data<MongoDBDriverReadConcernData>(v);
+	const char *rp_string = mongoc_read_concern_get_level(data->m_read_concern);
+	bson_t child;
+
+	bson_append_document_begin(bson, key, -1, &child);
+
+	bson_append_utf8(&child, "level", -1, rp_string, -1);
```

Comment:
It's not documented though... I checked that for john wage the other day, and therefore, reluctant to use it.